scraper/helper.ts

```diff
@@ -0,0 +1,147 @@
+import { Browser, Page } from 'puppeteer'
+import { WarningTag, ClassWarnings } from './interfaces'
+
+/**
+ * Remove any html character entities from the given string
+ * At this point, it only looks for 3 of them as more are not necessary
+ * @param { string } string The string to remove html characters from
+ * @returns { string }: string with html special characters replaced with english versions of said symbols
+ * @example
+ *    const clean = removeHtmlSpecials('&amp;') // 'and'
+ */
+const removeHtmlSpecials = (str: string) => {
+  // &amp --> and
+  let newstr = str.replace('&amp;', 'and')
+
+  // &nbsp ---> nothing (as it appears in course enrolment when the course does not have one)
+  newstr = newstr.replace('&nbsp;', '')
+
+  // &lt --> < (less than), this could be changed to before??
+  newstr = newstr.replace('&lt;', '<')
+
+  // There was no greater than sign found, but if neccessary, can be added here
+
+  return newstr
+}
+
+/**
+ * Converts dates into date objects
+ * @param { string[] } dateList: list of census dates to be formatted to utc time
```

Comment:
```suggestion
 * @param { string[] } dates: list of census dates to be formatted to utc time
```

When `someData` is a list you can just pluralise it instead of calling it a list

Reply:
Rename the actual param in the function as well!

Reply:
Although in this case it'll be more appropriate to be named as `dateStrings`

Reply:
Updated